src/Flag.js

```diff
@@ -13,29 +15,25 @@ const darkBorderColor = props => {
   }
 }
 
-const FlagShadow = styled('div')`
-  display: block;
-  width: 4px;
+const FlagShadow = styled(Box)`
   height: 4px;
   align-self: flex-end;
-  margin-right: -8px;
-  margin-bottom: -8px;
   border-width: 4px;
   border-style: solid;
   border-color: transparent;
   ${darkBorderColor};
+  position: absolute;
+  bottom: 0;
 `
 
-const FlagRight = styled('div')`
-  display: inline-block;
-  width: 18px;
+const FlagRight = styled(Box)`
   flex: none;
-  min-height: 32px;
-  margin-left: -10px;
   background-color: ${props => theme(`colors.${props.color}`)(props)};
-  border-radius: ${theme('radius')};
+  border-radius: 0 ${theme('radius')} ${theme('radius')} 0;
   /* for 32 x 8 triangle */
   transform: skew(-14deg);
+  position: relative;
+  z-index: 1;
```

Comment:
n-b: Are these z-indexes gonna be problematic if there are any z-index conflicts in the app that consumes this?

Reply:
We can also always update if it is an issue

Reply:
Great! Thanks :)